src/components/Body.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styled from 'styled-components';
+import EpisodesList from './EpisodesList';
+import GeneralInfo from './GeneralInfo';
+
+const BodyDiv = styled.div`
```

Comment:
A suggestion for these cases:

To call it a div can be problematic because we are coupling the implementation with the structure. If later we decide to change the tag to another one - change it for a `<main>` for example -  this name should be changed as well. 

In the app codebase we usually call this a "Wrapper" ("envoltório" in PT) or "Container".

Another possibility: since the main function of this component is to provid the Styled, we usually call this BodyStyled. Most times we move it to its own "Body.styled.js" if we feel the need to clean the component file a little bit.

Reply:
Actually, a common pattern is to move all the components focused on style to this separate file. You can find it useful when there is state logic mixed and this file starts getting bigger :grimacing: